package/Dockerfile

```diff
@@ -32,6 +34,9 @@ RUN mkdir -p /usr/share/harvester/harvester && \
     mkdir -p /usr/share/harvester/harvester/api-ui && \
     cd /usr/share/harvester/harvester/api-ui && \
     curl -sL https://releases.rancher.com/api-ui/${HARVESTER_API_UI_VERSION}.tar.gz | tar xvzf - --strip-components=1 && \
+    mkdir -p /usr/share/harvester/harvester/v1/harvester/plugin-assets && \
+    cd /usr/share/harvester/harvester/v1/harvester/plugin-assets && \
+    curl -sL https://releases.rancher.com/harvester-ui/plugin/harvester-${HARVESTER_UI_PLUGIN_BUNDLED_VERSION}.tar.gz | tar xvzf - --strip-components=1 && \
```

Comment:
@WuJun2016 do we need to keep the original UI assets, or only using the plugin UI would be enough?

Reply:
We still need the old address, which is for a single cluster UI, and the newly added address for multiple clusters